Function-Practice/script.js

```diff
@@ -0,0 +1,359 @@
+                                                    // PART 1
+
+//3 returnarea celui mai mic dintre numere
+
+const smallestNum = (num1, num2) => {
+  if (typeof num1 !== "number" || typeof num2 !== "number" || num1 === num2) {
+    return "One of them is not a number or they are equal"
+  }
+
+  return num1 < num2 ? `Smalest number is ${num1}` : `Smalest number is ${num2}`
+}
+
+const test = smallestNum(100, 100)
+// console.log(test)
+
+//5 de cate ori apare un caracter intr-un sir
+// Aici am trecut prin string asa deoarece este mai usor de scris. Ar trebuii sa fac un for normal in general?
+// Aici nu am vrut sa primeasca numere
+
+const countLetter = (string, letter) => {
+  //   if (
+  //     typeof string !== "string" ||
+  //     typeof letter !== "string" ||
+  //     !isNaN(letter)
+  //   ) {
+  //     return "This is not a word"
+  //   }
+
+  // Sunt putin dezorientat de ce am scris aici but it works
+
+  switch (true) {
```

Comment:
Merge pentru ca, ceea ce returneaza acel case  este o valoare de true/false si el o verifica cu true din switch. De aceea merge. Doar ca era mai ok cu if/else decat cu switch/case in acest caz